uninit_filter.py

```diff
@@ -26,10 +29,25 @@ def find_uninit_classes(coverage_file):
     return uninit_dict
 
 
+def output(uninit_dict, output_file):
+    if output_file is not None:
+        with open(output_file, 'w') as out:
+            for file in uninit_dict:
```

Comment:
I won't use `file` here, since its a builtin variable.  Notice how the syntax highlighting is different?  It's better practice to avoid clobbering builtins in your variable scope.

If you really want to use a builtin, just add an underscore after it.  `class` --> `class_`